dashboard/client/src/components/TitleCard.tsx

```diff
@@ -1,31 +1,32 @@
-import { Paper } from "@mui/material";
-import makeStyles from "@mui/styles/makeStyles";
+import { Box, Paper } from "@mui/material";
 import React, { PropsWithChildren, ReactNode } from "react";
 
-const useStyles = makeStyles((theme) => ({
-  card: {
-    padding: theme.spacing(2),
-    paddingTop: theme.spacing(1.5),
-    margin: theme.spacing(2, 1),
-  },
-  title: {
-    fontSize: theme.typography.fontSize + 2,
-    fontWeight: 500,
-    color: theme.palette.text.secondary,
-    marginBottom: theme.spacing(1),
-  },
-  body: {},
-}));
-
 const TitleCard = ({
   title,
   children,
 }: PropsWithChildren<{ title?: ReactNode | string }>) => {
-  const classes = useStyles();
   return (
-    <Paper className={classes.card} elevation={0}>
-      {title && <div className={classes.title}>{title}</div>}
-      <div className={classes.body}>{children}</div>
+    <Paper
+      sx={{
+        padding: 2,
+        paddingTop: 1.5,
+        margin: (theme) => theme.spacing(2, 1),
```

Comment:
nit:
```suggestion
        marginX: 1,
        marginY: 2,
```

Reply:
Good to use a uniform format, have modified them.